yt/frontends/ramses/data_structures.py

```diff
@@ -401,8 +401,11 @@ def __init__(self, filename, dataset_type='ramses',
             # note: we exclude the unlikely event that one of the group is actually a file
             # instad of a folder
             self.num_groups = len(
-                filter(lambda e: os.path.isdir(e),
-                       glob(os.path.join(root_folder, 'group_?????'))))
+                list(
+                    filter(lambda e: os.path.isdir(e),
```

Comment:
I know this isn't new, but any chance we could rework this to a list comprehension?

`[_ for _ in glob(os.path.join(root_folder, 'group_?????')) if os.path.isdir(_)]`